extra-requirements.txt

```diff
@@ -72,7 +72,7 @@ jsonschema:                 cicd
 dgl:                        cicd
 black==20.8b1:              test
 matplotlib:                 devel
-kubernetes>=18.20.0:        devel
+kubernetes>=18.20.0:        standard, devel
```

Comment:
if it is standard, u can remove from `devel` I'd say

Reply:
actually it fails if I remove it? like here https://github.com/jina-ai/jina/pull/3275/checks?check_run_id=3719205391

Reply:
also it seems that everything in standard is also in devel atm? (and daemon but I think thats not necessary here)

Reply:
not sure that works? all other requirements are basically standard, daemon, devel?

Reply:
is our k8s support mature enough to be in `standard`? its docs is still in the experimental section now.

suggest remove `standard`, keep `devel`

Reply:
This relates to a problem raised in hubble slack channel. Kubernetes is now needed in the hub executors.

Reply:
> Kubernetes is now needed in the hub executors.
what does this mean? do you mean Hub executor should do `pip install kubernetes`  in each Dockerfile?